okhttp/src/jvmMain/kotlin/okhttp3/OkHttpClient.kt

```diff
@@ -231,6 +231,12 @@ open class OkHttpClient internal constructor(
   @get:JvmName("minWebSocketMessageToCompress")
   val minWebSocketMessageToCompress: Long = builder.minWebSocketMessageToCompress
 
+  /**
+   * Maximum outbound web socket message size (in bytes) that will be sent
```

Comment:
nit: It's not the limit for a single message, rather the queue.

Previous docs

``` 
     * The maximum number of bytes to enqueue. Rather than enqueueing beyond this limit we tear down
     * the web socket! It's possible that we're writing faster than the peer can read.
```

Reply:
I restored the previous documentation 👍